culture.md

```diff
@@ -54,9 +54,9 @@ The rule in the Code of Conduct about avoiding discussions about identity labels
 
-It is a policy of this community that people's own identities (and corresponding labels) must be respected as presented, and since there did not seem to be a clear way to prevent the cycle of challenging and relitigating the issue, a decision was made to ban that, and any such discussion -- this now includes any discussion related to trans vs trans\*, lesbian vs bi, 'gay' and 'queer' as generics, etc. Note that this only applies for self-identification. Forcibly applying a label that you think is fitting to someone who does not self-identify that way is still considered harassment as per our Code of Conduct.
+It is a policy of this community that people's own identities (and corresponding labels) must be respected as presented, and since there did not seem to be a clear way to prevent the cycle of challenging and relitigating the issue, a decision was made to ban any discussion about someone else's identity. This also includes discussion of other identities as they relate to your own ("I am pan because I am not bi, which means...") -- this policy extends to other identities, such as any discussion related to trans vs trans\*, lesbian vs bi, 'gay' and 'queer' as generics, etc. Note that this only applies for self-identification. Forcibly applying a label that you think is fitting to someone who does not self-identify that way is still considered harassment as per our Code of Conduct. All discussions on gender and sexuality contain the expectation that the participants will stay in their own "lane" as it were, and speak only on their own experiences.
 
-If you are unsure what a label means, feel free to research it externally or, if the person who used the label clearly consents to the conversation, ask them *in private* for a simple clarification, but understand that the CoC will most likely not favor you if the conversation goes awry. *Do not have these discussions in open channels*. Because of their inherent complexity, it is too easy to step on toes even if you don't mean to, in very serious, hurtful ways.
+If you are unsure what a label means, feel free to research it externally (there are plenty of people discussing the intricacies of their labels in exhaustive detail via publicly viewable blogs and thinkpieces) or, if the person who used the label clearly consents to the conversation, ask them *in private* for a simple clarification, but understand that the CoC will most likely not favor you if the conversation goes awry. *Do not have these discussions in open channels*. Because of their inherent complexity, it is too easy to step on toes even if you don't mean to, in very serious, hurtful ways. Some channels explicitly allow discussion of labels as it pertains to self identification (#what_am_i) or other exceptions to this policy, but they are responsible for calling out those exceptions explictly, and all other expectations around harrassment and listening to requests for redirects still apply.
 
```

Comment:
The "feel free to" carries a very dismissive to, at least to me. I also thing putting the options into a list would make them more digestible, as would splitting the para:

---- 

If you are unsure what a label means, there are a few options:

* research it externally (there are plenty of people discussing the intricacies of their labels in exhaustive detail via publicly viewable blogs and thinkpieces), or
* if the person who used the label clearly consents to the conversation, ask them *in private* for clarification (but understand that the CoC will most likely not favor you if the conversation goes awry.)

But in any case: *do not have these discussions in open channels*. Because of their inherent complexity, it is too easy to step on toes even if you don't mean to, in very serious, hurtful ways.

Some specific channels may explicitly allow discussion of labels as it pertains to self identification (e.g., `#what_am_i`) or other exceptions to this policy. However, those channels are responsible for calling out those exceptions explicitly. All other expectations around harassment and listening to requests for redirects still apply.